app/src/pages/inside/testsPage/historyLine/historyLineItem/historyLineItemContent/historyLineItemBadges/historyLineItemBadges.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import {
+  AUTOMATION_BUG,
+  NO_DEFECT,
+  PRODUCT_BUG,
+  SYSTEM_ISSUE,
+  TO_INVESTIGATE,
+} from 'common/constants/defectTypes';
+import { SKIPPED, RESETED, FAILED, MANY, NOT_FOUND } from 'common/constants/launchStatuses';
+import Parser from 'html-react-parser';
+import styles from './historyLineItemBadges.scss';
+import NoItemIcon from './img/noItem-inline.svg';
+import EmptyItemIcon from './img/emptyItem-inline.svg';
+import NotEyeIcon from './img/notEyeItem-inline.svg';
+import DurationIcon from './img/duration-icon-inline.svg';
+import InfoIcon from './img/info-icon-inline.svg';
+
+const cx = classNames.bind(styles);
+
+const defectsTitleMap = {
+  [AUTOMATION_BUG]: 'ab',
+  [NO_DEFECT]: 'nd',
+  [PRODUCT_BUG]: 'pb',
+  [SYSTEM_ISSUE]: 'si',
+  [TO_INVESTIGATE]: 'ti',
+};
+
+export class HistoryLineItemBadges extends Component {
+  static propTypes = {
+    active: PropTypes.bool,
+    status: PropTypes.string,
+    issue: PropTypes.object,
+    growthDuration: PropTypes.string,
+    defects: PropTypes.object,
+  };
+
+  static defaultProps = {
+    active: false,
+    status: '',
+    issue: {},
+    growthDuration: '',
+    defects: {},
+  };
+
+  getDefectBadge = () => {
+    const { defects } = this.props;
+    const defectType = Object.keys(defects).find((key) => defects[key].total);
+
+    return (
+      defectType && (
+        <div key={defectType} className={cx('defect-badge', defectsTitleMap[defectType])}>
+          <span>{defectsTitleMap[defectType]}</span>
+        </div>
+      )
+    );
+  };
+
+  getBadges = () => {
+    const { status, growthDuration } = this.props;
+    const badges = [];
+
+    switch (status.toLowerCase()) {
+      case FAILED: {
+        badges.push(this.getDefectBadge());
+        break;
+      }
+      case SKIPPED: {
```

Comment:
You can merge FAILED and SKIPPED options

Reply:
Fixed